src/lib/FieldMapper/ContentFieldMapper/UserDocumentFields.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Solr\FieldMapper\ContentFieldMapper;
+
+use eZ\Publish\SPI\Persistence\Content as SPIContent;
+use eZ\Publish\SPI\Search\Field;
+use eZ\Publish\SPI\Search\FieldType;
+use EzSystems\EzPlatformSolrSearchEngine\FieldMapper\ContentFieldMapper;
```

Comment:
This is legacy namespace within the package.

Reply:
Fixed in c0bc7e5 (forced push)